content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors: <a href="#Junior-accounts">Junior accounts</a> and <a href="#Adult-accounts">Adult accounts for users between 14 and 16 years old</a>. They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
+
+<h2 id="Junior-accounts"></h2>
```

Comment:
```suggestion
<a id="Junior-accounts"></a>
```
I'm not sure if either is considered good practise but this is what I've been using to add "anchor targets".